data/card_catalogue/npc/npc-generic.json

```diff
@@ -5,49 +5,36 @@
     "tier": "blue",
     "details": {
       "rules": {
-        "active": "You can use an additional ranged attack.",
+        "active": "Make an instant attack that cannot miss.",
```

Comment:
Adopting the new wording used in double strike. Also just using 'attack' instead of ranged attack, since I don't think it really matters what kind of attack they make.